setup.py

```diff
@@ -10,6 +11,25 @@
 with open('CHANGES.rst') as f:
     changes = f.read()
 
+install_requires = ['pycodestyle==2.3.1', 'pyflakes==1.6.0',
+                    'pyserial==3.4', 'pyqt5==5.10.1', 'qscintilla>=2.10',
+                    'qtconsole==4.3.1', 'matplotlib==2.1.2',
+                    'pgzero==1.2', 'PyQtChart>=5.10', 'appdirs>=1.4.3',
+                    'gpiozero>=1.4.1', 'guizero>=0.4.5',
+                    'pigpio>=1.40.post1', 'Pillow>=5.0.0',
+                    'requests>=2.18.4']
+
+# Exclude packages not available for ARM in PyPI/piwheels (Raspberry Pi)
+try:
+    machine = platform.machine()
+    if machine.lower().startswith('arm'):
+        exclude = ('pyqt5', 'qscintilla', 'qtconsole', 'PyQtChart')
+        install_requires = [requirement for requirement in install_requires
+                            if not requirement.startswith(exclude)]
+except Exception as e:
+    # Something unexpected happened, so simply keep all requires
+    pass
+
```

Comment:
This is a bit ugly to have in the setup.py, but I found it to be the best solution to dynamically chose what to include in the `install_requires`. Otherwise mu-debug doesn't work.
Using something like extra_requires is a lot messier during installation.